packages/superset-ui-connection/src/types.ts

```diff
@@ -33,7 +33,7 @@ export type Method = RequestInit['method'];
 export type Mode = RequestInit['mode'];
 export type Redirect = RequestInit['redirect'];
 export type ClientTimeout = number | undefined;
-export type ParseMethod = 'json' | 'text' | null;
+export type ParseMethod = 'json' | 'text' | 'raw' | null | undefined;
```

Comment:
Had to allow `undefined`, too, otherwise `parseResponse(apiPromise, undefined)` (with specific `undefined` argument) will return `Promise<JsonResponse | TextResponse | Response>` instead of `Promise<ResultType<T>>` we want.